.github/workflows/publish.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+concurrency:
+  group: publish
+  cancel-in-progress: true
+permissions: write-all
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          cache: npm
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
```

Comment:
again. still don't have any package lock file. so it complained again on the workflow action
using `npm install` would solve it.

Reply:
i have changed it